susemanager/src/mgr-create-bootstrap-repo

```diff
@@ -585,9 +587,16 @@ def create_repo(label, options, mgr_bootstrap_data, additional=[]):
             rel_path = os.path.relpath(file_path, start=destdirtmp)
             (l_path, ext) = rel_path.rsplit('.', 1)
             if ext:
+                dg = hashlib.sha256()
+                with open(file_path, "rb") as pkg_fh:
+                    while True:
+                        buff = pkg_fh.read(0x1000)
+                        if not buff:
+                            break
+                        dg.update(buff)
```

Comment:
Just a suggestion here to make this cleaner:

```suggestion
                dg = hashlib.sha256(open(file_path,'rb').read())
```

Reply:
It's cleaner, but will read whole file at once and it's ~20-30Mb, it shouldn't be a problem but is it a really good practice?

Reply:
Hmmm, I see your point. I don't think 20-30Mb will cause any problem here, as the file is not really that big, but since now I see the reason why you implemented that way, I am perfectly fine if we just keep the code as you did :)

Thanks!

Reply:
Calling `open()` outside of a `with` statement and without calling `close()` later on is not cleaner IMO. I prefer Victor's approach even if everything is read at once with `.read()`.